node/typescript/src/ampli/index.ts

```diff
@@ -580,6 +569,31 @@ export class Ampli {
     return { promise };
   }
 
+  groupIdentify(
+    groupType: string,
+    groupName: string,
```

Comment:
Can node take string[] as groupName?

Reply:
NVM, you are right, node only allows string as gropuName.
https://github.com/amplitude/Amplitude-Node/blob/a3f6b173a6884cb11168b6d09ae3d0351533c34a/packages/identify/src/identify.ts#L54